src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -138,32 +141,37 @@ void ColorScheme::LayerJson(const Json::Value& json)
     }
 }
 
-std::wstring_view ColorScheme::GetName() const noexcept
+winrt::hstring ColorScheme::Name() const noexcept
 {
-    return { _schemeName };
+    return { _schemeName.c_str() };

```

Comment:
hol up, what

Reply:
don't do this. if you're ever going from one string type to another string type, _don't use the data pointer unless absolutely necessary_.

winrt::hstring has a constructor that just takes a wstring. just use that one.